build.gradle

```diff
@@ -10,11 +6,23 @@ buildscript {
   }
 
   dependencies {
-    classpath 'com.android.tools.build:gradle:7.1.1'
+    classpath 'com.android.tools.build:gradle:7.2.0-alpha07'
     classpath 'io.github.gradle-nexus:publish-plugin:1.1.0'
   }
 }
 
+plugins {
+  id 'com.android.application' version '7.2.0-alpha07' apply false
+
+  /* TODO: Re-evaluate the need to have this plugin declared at the top-level build.gradle.  With
+      this plugin only applied at the module-level, the Compose compiler fails with a very low level
+      Kotlin IR error, so we're keeping it here for now. gkotula 2022-01-13 */
+  id 'org.jetbrains.kotlin.android' version '1.6.10' apply false
```

Comment:
I just checked again today whether this needs to be at the top-level, and the build still fails if this is moved to the new template module's `build.gradle`.  So I guess it has to stay here for now.